modules/databases/download_references.nf

```diff
@@ -7,6 +7,6 @@ process download_references {
     script:
         """
         git clone https://github.com/mult1fractal/WtP_phage_reference_db.git
-        cat WtP_phage_reference_db/*.fa > phage_references.fa
+        cat WtP_phage_reference_db/fasta-files/*.fa > phage_references.fa
```

Comment:
this might break older "workflow environments" (e.g. -r 0.7 will use the old cat *.fa) 
so you should reverse that reference git. and create a new one with a clear structure!!!! (think about how to structure it

Reply:
so for the future the new reference git should use "tags" which the workflow is pulling